pynicotine/config.py

```diff
@@ -708,6 +711,13 @@ def set_config(self):
             if shared_folder not in shares and virtual_name not in (x[0] for x in shares):
                 shares.append(shared_folder)
 
+        # Migrate old trusted buddy shares to new format
```

Comment:
`#` ... `since >= 3.3.0`

Reply:
I'll add version numbers to all migration comments later.